estate/models/estate_property.py

```diff
@@ -3,17 +3,15 @@
 
 
 def _default_date_availability(self):
-        return fields.Date.context_today(self) + relativedelta(months=3)
+        return fields.Date.context_today(self) 
 
 class EstateProperty(models.Model):
     _name="estate.property"
     _description="Test model prop"
     name = fields.Char("Title", required=True)
     description = fields.Text("Description")
     postcode = fields.Char("Postcode")
-
-
-    
+    User_Id = fields.Char("Postcode")    
```

Comment:
snake_case
Et "Postcode" ne me semble pas être une valeure appropriée ni le type `fields.Char`
Exemple:
```suggestion

    user_Id = fields.Many2one("res.users", string="Seller")    
```

Reply:
D'ACCORD JE VOIS MERCI!